### PR TITLE
Add list test to simplify template

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR introduces the following tests
- is_list

And implements it in the plugins.rst.js2 template.

**Update:** Simplified version

##### ISSUE TYPE
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
module docs

##### ANSIBLE VERSION
v2.5

### CODE REVIEW

**Comment by dagwieers** [comment]
Unfortunately there is no simple way in Jinja2 to test if value is a boolean, the common practice seems to be to convert it into a string and compare it to 'False' or 'True'. WFT!

I think this is an important omission in the default Jinja2 tests...

**Comment by dagwieers** [comment]
@dharmabumstead Can we get this merged ?

**Comment by sivel** [comment]
Why was this merged without approval? I don't see that @dharmabumstead ever approved this.

We don't just merge things because we believe reviews are taking too long.

**Comment by dharmabumstead** [comment]
We also didn’t get reviews from @abadger or @gundalow.

**Comment by dagwieers** [comment]
It has been breaking docs for the past 4 days, so I'd like to make sure v2.5 docs build fine when it gets released. The changes are trivial.
screenshot from 2018-03-20 23-26-09

File: docs/bin/plugin_formatter.py [hunk 1]
```diff
@@ -132,6 +132,12 @@ def rst_xline(width, char="="):
     return char * width
 
 
+# NOTE: The existing Jinja2 'sequence' tests matches strings and dictionaries
+def test_list(value):
+    ''' Return true if objects is a list or tupple '''
+    return isinstance(value, Sequence)
```
Comment (webknjaz) [inline docs/bin/plugin_formatter.py:138]: @dagwieers `str` is also `Sequence`. So it's either docstring or the check is misleading.
You might want to add `and not isinstance(value, str)`

Comment (dagwieers) [reply docs/bin/plugin_formatter.py:138]: Yup, I know: https://github.com/ansible/ansible/pull/37517/files#diff-bc6bf5732a9c00dcb3cb182f4ce3b7c8R100

But that's no longer my problem now, is it :-)

**Pull request merged**